requirements.txt

```diff
@@ -24,32 +24,31 @@ yq==2.4.1
 ## The following requirements were added by pip freeze:
 asn1crypto==0.24.0
 backoff==1.0.7
-boto3==1.4.4
+boto3==1.4.8
 botocore==1.8.35
-certifi==2018.1.18
+certifi==2018.4.16
 cffi==1.11.5
 chardet==3.0.4
-click==6.7
 colorama==0.3.7
 contextlib2==0.4.0
 cryptography==1.9
 docopt==0.6.2
 docutils==0.14
-Flask==0.12.2
+Flask==0.10.1
```

Comment:
A downgrade feels weird - intentional?

Reply:
We're pinning Flask to 0.10.1 in all the FE apps - I wonder why scripts was higher to begin with?

Reply:
Yeah, I hadn't noticed that TBH. It's the pin in utils which is dictating the version here.

Reply:
Oh, I think it'll have been when @samuelhwilliams fixed it from not being pinned at all, he probably picked the latest non-1.0 version? I could have that completely wrong. Consistency with the frontends makes sense, as long as nothing now breaks as a result :-)